src/js/angular/clustermanagement/directives/cluster-list.directive.js

```diff
@@ -1,4 +1,16 @@
-import {Location} from "../../models/clustermanagement/cluster";
+import {ClusterConfiguration, Location} from "../../models/clustermanagement/cluster";
+
+const getAdvancedOptionsClass = function () {
```

Comment:
Why is this function declared here, and later a variable is created with a reference to it? Why not simply create a public function `getAdvancedOptionsClass` within the directive? By making it a public function, you can use the second argument of the link method, `element`, to check if there is an `optionsModule`.

Reply:
done